PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!-- Content enclosed in HTML comments will not be rendered in the Markdown, and are intended to help guide you -->
+
+<!-- This PR template is inherited from https://github.com/opendoor-labs/.github -->
+
+## Context
+
+> Why are you making this change?
+
+## Origin
+
+> What team/individual requested this change? Please link to a JIRA ticket or other documentation if applicable. If you identified this issue yourself please write "Me" (and still link to any additional context if it exists).
+
+## Summary of Changes
+
+<!-- Provide a short description or bulleted list of changes made in this PR. -->
+
+## Test Plan
+
+<!-- How have you tested this change, and what further testing will be done? What’s the riskiest part of this PR? How will you test and monitor that? -->
+
+## Documentation
+
+<!-- This is a task list. To mark a task as complete, use [x]. See https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#task-lists -->
+
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to external documentation if necessary (READMEs, Confluence, etc.)
```

Comment:
Here's an example of a task list on an actual PR: https://github.com/markrmullan/.github/pull/4